README.md

```diff
@@ -337,6 +337,7 @@ Name | Website | Region
 [PeopleDoc](/company-profiles/peopledoc.md) ⚠️️ | http://www.people-doc.com |
 [Percona](/company-profiles/percona.md) | https://www.percona.com | Worldwide
 [PicksCo](/company-profiles/picksco.md) | http://interchangehq.com/ | UTC-7 to UTC+1
+[Platform Builders](/company-profiles/platform-builders.md) | https://platformbuilders.io/ | Brazil and Portugal
```

Comment:
The "Region" in this section should match the region in your company profile. 

You mention Worldwide there so if you hire people worldwide, please update this to match. Thanks!

Reply:
Ok. Change the region of Platform Builders to Worldwide. Thanks.